README.rst

```diff
@@ -5,58 +5,47 @@
 Overview
 --------
 
-Cedar is a collection of data-focused tools for data generated by and
-used in Evergreen builds. It is not a part of evergreen, but a
-companion service and tool kit. In particular, Cedar focuses on the
-"offline" data processing cases, and providing access to data to
-support external user interfaces (e.g. cli tools, data visualizations,
-etc.)
-
-Build operations generate data, and Evergreen stores most of this data
-directly: the outcome of tasks, the outcomes of specific tests,
-artifacts produced by builds. There's other data, specific to
-particular projects, data that is less naturally structured or less
-specifically linked to specific execution events. Additionally, cedar
-provides a platform to collect and analyze data *about* an evergreen
-deployment in the large.
+Cedar is a collection of data-focused tools for data generated by and used in Evergreen builds. It is not a part of
+Evergreen, but serves as a companion service and tool kit. In particular, Cedar focuses on the "offline" data processing
+cases, and providing access to data to support external user interfaces (e.g. cli tools, data visualizations, etc.)
+
+Build operations generate data, and Evergreen stores most of this data directly (e.g. the outcome of tasks, artifacts
+produced by builds). There's other data, specific to particular projects, that is less naturally structured or less
+specifically linked to specific execution events. Additionally, Cedar provides a platform to collect and analyze data
+*about* an Evergreen deployment in the large.
 
 Components
 ~~~~~~~~~~
 
-Cedar has three core components:
+- A simple JSON-only REST web service to data storage and access (see the ``rest`` package).
 
-- a simple JSON-only REST web service to data storage and access. (See
-  the ``rest`` sub-package.)
+- A gRPC service as an alternative to the REST API, which also supports data storage and access (see the ``rpc``
```

Comment:
This actually does not support access, just data storage.

Reply:
Fixed.